build.sbt

```diff
@@ -2,35 +2,29 @@ lazy val akkaHttpVersion = "10.1.5"
 lazy val akkaVersion    = "2.5.16"
 lazy val sparkVersion = "2.3.2"
 
-lazy val root = (project in file(".")).
-  settings(
-    inThisBuild(List(
-      organization    := "com.example",
-      scalaVersion    := "2.11.12"
-    )),
-    name := "db2eventstoreakkastreams",
-    libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-http"            % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-http-xml"        % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-stream"          % akkaVersion,
+name := "db2eventstoreakkastreams"
+organization    := "com.example"
+scalaVersion    := "2.11.12"
 
-      "com.typesafe.akka" %% "akka-http-testkit"    % akkaHttpVersion % Test,
-      "com.typesafe.akka" %% "akka-testkit"         % akkaVersion     % Test,
-      "com.typesafe.akka" %% "akka-stream-testkit"  % akkaVersion     % Test,
-      "com.novocode" % "junit-interface" % "0.11" % Test,
-      "org.scalatest"     %% "scalatest"            % "3.0.1"         % Test
-    ),
-    libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-slf4j" % "2.5.17",
-      "ch.qos.logback" % "logback-classic" % "1.2.3"
-    ),
-    libraryDependencies ++= Seq(
-      "com.lightbend.akka" %% "akka-stream-alpakka-csv" % "0.18",
+libraryDependencies ++= Seq(
+  "com.typesafe.akka" %% "akka-http"                % akkaHttpVersion,
+  "com.typesafe.akka" %% "akka-http-spray-json"     % akkaHttpVersion,
+  "com.typesafe.akka" %% "akka-http-xml"            % akkaHttpVersion,
+  "com.typesafe.akka" %% "akka-actor"               % akkaVersion,
+  "com.typesafe.akka" %% "akka-protobuf"            % akkaVersion,
+  "com.typesafe.akka" %% "akka-stream"              % akkaVersion,
+  "com.typesafe.akka" %% "akka-slf4j"               % akkaVersion,
```

Comment:
A few of these were transitively pulled in with older versions from some other dependency, Akka module versions must never be mixed.